geonode/static/gruntfile.js

```diff
@@ -117,11 +125,11 @@ module.exports = function(grunt) {
             'datatables/media/js/jquery.dataTables.js',
             'jquery-timeago/jquery.timeago.js',
             'tinysort/src/jquery.tinysort.js',
-            'raty/lib/jquery.raty.min.js',
-            'jquery-waypoints/waypoints.min.js',
-            'jquery-ui/ui/minified/jquery-ui.custom.min.js',
+            'raty/lib/jquery.raty.js',
+            'jquery-waypoints/waypoints.js',
+            'jquery-ui/ui/jquery-ui.custom.js',
```

Comment:
I need to change these, because it is what being referenced in `base.html` when DEBUG_STATIC = True.
The developers probably need the unminified version for debugging.